### PR TITLE
Removed useless back to top links

### LINKED ISSUE #15431: Remove useless back to top links
There are some pages that has Back to top links, see [current search results])https://github.com/dotnet/docs/search?q=Back+to+top&type=Code).

I believe they existed in an old system that didn't have the current side bar that makes it easy to navigate in the article. They're not needed now.

### PR DESCRIPTION
**Pull request by srhsrhsrhsrh**
## Summary

Removed all instances of "[Back to top](#top)" from project 

Fixes #15431

### CODE REVIEW

File: docs/framework/interop/interop-marshaling.md
Comment (pkulikov) [inline docs/framework/interop/interop-marshaling.md]: With that PR, all `<a name="top"></a>` at the beginning of the files become useless.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @sarahliii for creating this PR to remove those legacy back to top links. @pkulikov is right that we should remove the anchor from the top as well. 

I'd even take a step further and remove the menu on the top and the additional anchors from those articles since that's already generated in the new system. But I'd leave it up to you whether you'd like to address this on this PR or not.

**Comment by sarahliii** [comment]
> Thanks @sarahliii for creating this PR to remove those legacy back to top links. @pkulikov is right that we should remove the anchor from the top as well.
> 
> I'd even take a step further and remove the menu on the top and the additional anchors from those articles since that's already generated in the new system. But I'd leave it up to you whether you'd like to address this on this PR or not.

I'll work on removing the anchor for "top". I think I'll hold off on the further changes since I'm not so familiar with the system. (Wouldn't want to break anything!)

**Comment by mairaw** [comment]
Yeah including the intro sentence. Here’s an example:
722C226F-A559-4EC3-92AB-6F8BA8E502FD

But it’s totally fine. Let’s keep this one focused on the task at hand.

Don’t be afraid though! You shouldn’t be able to break anything and we can always revert PRs as well. Thank you!

**Comment by mairaw** [comment]
And I wouldn’t remove the anchors wholesale. We can clean some that seem useless and see if we get any build warnings.

File: docs/framework/misc/security-transparent-code-level-2.md
Comment (mairaw) [inline docs/framework/misc/security-transparent-code-level-2.md]: Remove the top anchor from this file too please

File: docs/framework/misc/security-transparent-code.md
Comment (mairaw) [inline docs/framework/misc/security-transparent-code.md]: Same on this file

File: docs/framework/performance/interop-etw-events.md
Comment (mairaw) [inline docs/framework/performance/interop-etw-events.md]: I’d roll back this change

File: docs/framework/performance/method-etw-events.md
Comment (mairaw) [inline docs/framework/performance/method-etw-events.md]: Still has the anchor tag as well

File: docs/framework/performance/security-etw-events.md
Comment (mairaw) [inline docs/framework/performance/security-etw-events.md]: Please roll back this change

File: docs/standard/security/cryptographic-services.md
Comment (mairaw) [inline docs/standard/security/cryptographic-services.md]: Please undo this change

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @sarahliii. There are still a few things to address before we can merge this.

**Comment by nemrism** [comment]
I'm surprised this subject is not on @NextTurn bucket list ;)
Nice 1st contribution @sarahliii

**Comment by mairaw** [comment]
I have resolved some merge conflicts that showed up during the day since they were complicated to resolve. Make sure to do git pull before you push any changes. Also, please let us know if you need any help to finish this PR. Thanks!

**Comment by NextTurn** [comment]
@nemrism I've got a bunch of other things to work on recently 😅.

**Comment by nemrism** [comment]
> @nemrism I've got a bunch of other things to work on recently 😅.

I'm also still trying to design a better solution to work on the #10653 subject!

**Review by mairaw — APPROVED** [review]
I've addressed my final feedback so we can merge this. Thanks @sarahliii!

**Pull request merged**
